### PR TITLE
Em 3834 fix smtp settings

### PR DESCRIPTION
**Pull request by mdayican**
### Change description ###

Em 3834 fix smtp settings

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java [hunk 1]
```diff
@@ -31,13 +33,19 @@ public SummaryReportService(
         HearingRecordingStorage hearingRecordingStorage
     ) {
         this.emailSender = emailSender;
-        this.recipients = recipients;
+        if (recipients == null || recipients.length == 0) {
+            throw new RuntimeException("No recipients configured for reports");
```
Comment (apereverzin) [inline src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java:37]: I assume this should be discussed with the business

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java:37]: to send email we need recipients and task omits exceptions

Comment (apereverzin) [reply src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java:37]: is it possible not to send report if there are no recipients defined? or not sending reports means application should not run?

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/em/hrs/service/email/SummaryReportService.java:37]: report sending task is configurable if it is enabled we need to make this error visible .

**Review by apereverzin — APPROVED** [review]

**Pull request merged**